src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/SocketsHttpHandler.cs

```diff
@@ -376,7 +402,8 @@ private Exception ValidateAndNormalizeRequest(HttpRequestMessage request)
                 // HTTP 1.0 does not support chunking
                 if (request.Headers.TransferEncodingChunked == true)
                 {
-                    return new NotSupportedException(SR.net_http_unsupported_chunking);
```

Comment:
I think we should still throw an exception if a user requests both chunked and HTTP/1.0.

Reply:
Reverted to that functionality